docs/command-line/promtool.md

```diff
@@ -632,6 +632,15 @@ Import samples from OpenMetrics input and produce TSDB blocks. Please refer to t
 
 
 
+###### Flags
+
+| Flag | Description |
+| --- | --- |
+| <code class="text-nowrap">--label</code> | Label to attach to metrics. Can be specified multiple times. |
```

Comment:
Can we show the name=value syntax?  I don't think people will guess.
Like you did in the PR description - `--label=cluster_name=demo --label=user=test`

Reply:
Note the docs are generated, so change in the flag description in the code.

Reply:
@bboreham  Added the `--label` example in the help command. 
@dgl Yes, I am auto generating the docs using `make cli-documentation`